test/proxy_funds.js

```diff
@@ -38,82 +38,125 @@ contract('Proxy funds', accounts => {
+  const recoverEth = async (target, vault) => {
     const amount = 1
-    const initialBalance = await getBalance(proxy.address)
+    const initialBalance = await getBalance(target.address)
     const initialVaultBalance = await getBalance(vault.address)
-    const r = await proxy.sendTransaction({ value: 1, gas: 31000 })
-    assert.equal((await getBalance(proxy.address)).valueOf(), initialBalance.plus(amount))
-    await proxy.transferToVault(ETH)
-    assert.equal((await getBalance(proxy.address)).valueOf(), 0)
+    const r = await target.sendTransaction({ value: 1, gas: 31000 })
+    assert.equal((await getBalance(target.address)).valueOf(), initialBalance.plus(amount))
+    await target.transferToVault(ETH)
+    assert.equal((await getBalance(target.address)).valueOf(), 0)
     assert.equal((await getBalance(vault.address)).valueOf(), initialVaultBalance.plus(initialBalance).plus(amount).valueOf())
   }
 
-  const recoverTokens = async (proxy, vault) => {
+  const recoverTokens = async (target, vault) => {
     const amount = 1
     const token = await getContract('StandardTokenMock').new(accounts[0], 1000)
-    const initialBalance = await token.balanceOf(proxy.address)
+    const initialBalance = await token.balanceOf(target.address)
     const initialVaultBalance = await token.balanceOf(vault.address)
-    await token.transfer(proxy.address, amount)
-    assert.equal((await token.balanceOf(proxy.address)).valueOf(), initialBalance.plus(amount))
-    await proxy.transferToVault(token.address)
-    assert.equal((await token.balanceOf(proxy.address)).valueOf(), 0)
+    await token.transfer(target.address, amount)
+    assert.equal((await token.balanceOf(target.address)).valueOf(), initialBalance.plus(amount))
+    await target.transferToVault(token.address)
+    assert.equal((await token.balanceOf(target.address)).valueOf(), 0)
     assert.equal((await token.balanceOf(vault.address)).valueOf(), initialVaultBalance.plus(initialBalance).plus(amount).valueOf())
   }
 
-  const failWithoutVault = async (proxy, vault) => {
+  const failWithoutVault = async (target, vault) => {
     const amount = 1
     const vaultId = hash('vaultfake.aragonpm.test')
-    const initialBalance = await getBalance(proxy.address)
-    await kernel.setDefaultVaultId(vaultId)
-    const r = await proxy.sendTransaction({ value: 1, gas: 31000 })
-    assert.equal((await getBalance(proxy.address)).valueOf(), initialBalance.plus(amount))
+    const initialBalance = await getBalance(target.address)
+    await kernel.setRecoveryVaultId(vaultId)
+    const r = await target.sendTransaction({ value: 1, gas: 31000 })
+    assert.equal((await getBalance(target.address)).valueOf(), initialBalance.plus(amount))
     return assertRevert(async () => {
-      await proxy.transferToVault(ETH)
+      await target.transferToVault(ETH)
     })
   }
 
-  context('App Proxy', async () => {
+  context('App without proxy is not recoverable', async () => {
```

Comment:
Maybe the whole picture is still not clear enough for me, but do we really want this? At least is kind of confusing that `AragonApp is VaultRecoverable`.
Shouldn't we have a test for app without kernel?

Reply:
Oops, should've redone this part as "App without kernel".